api/v1alpha1/releaselink_webhook.go

```diff
@@ -29,12 +30,19 @@ func (r *ReleaseLink) SetupWebhookWithManager(mgr ctrl.Manager) error {
 		Complete()
 }
 
-// +kubebuilder:webhook:path=/mutate-appstudio-redhat-com-v1alpha1-releaselink,mutating=true,failurePolicy=fail,sideEffects=None,groups=appstudio.redhat.com,resources=releaselinks,verbs=create;update,versions=v1alpha1,name=mreleaselink.kb.io,admissionReviewVersions=v1
+// +kubebuilder:webhook:path=/mutate-appstudio-redhat-com-v1alpha1-releaselink,mutating=true,failurePolicy=fail,sideEffects=None,groups=appstudio.redhat.com,resources=releaselinks,verbs=create,versions=v1alpha1,name=mreleaselink.kb.io,admissionReviewVersions=v1
 
 var _ webhook.Defaulter = &ReleaseLink{}
 
 // Default implements webhook.Defaulter so a webhook will be registered for the type.
 func (r *ReleaseLink) Default() {
+	label := "release.appstudio.openshift.io/auto-release"
```

Comment:
I don't like this hard-coding here. We should revamp it moving `release.appstudio.openshift.io` labels to a single place. Right now we have something like that in the tekton package but that doesn't look like the right place after this change. Having something like this in, for example, the Release type as exported constants, would reduce the hard-coding in the integration service as well, which would use our constant directly.

Having said this, I don't want to address this at this time. I'll create a ticket for it.